charts/osm/templates/osm-deployment.yaml

```diff
@@ -73,6 +83,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+          securityContext:
+            allowPrivilegeEscalation: false
```

Comment:
please remove this and make it a separate change with more context (not disputing the change but rather why it is being done in this pr).